docs-ref-conceptual/get-started-tutorial-3-use-variables.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Get variable values from Azure resources or a local JSON file | Microsoft Docs
+description: Learn how to use variables to store JSON file properties and query output.
+manager: jasongroce
+author: dbradish-microsoft
+ms.author: dbradish
+ms.date: 11/15/2023
+ms.topic: conceptual
+ms.service: azure-cli
+ms.tool: azure-cli
+ms.custom: devx-track-azurecli
+keywords: azure, 
+---
+# Populate variables for use in scripts
+
+In this tutorial step you will learn how to get information from a local JSON file, new and existing Azure resources, and store the information in a variable. The variable can then be used in Azure CLI to perform build and destroy jobs at scale. Some common use-cases are:
```

Comment:
I think Acrolinx usually likes things in present tense, consider changing "you will learn" to "you learn"

Reply:
Add a comma after "In this tutorial" for better flow

Reply:
I just ran Acrolinx on each file.  Updates forthcoming.